docs/migrating-from-0.9.md

```diff
@@ -0,0 +1,251 @@
+# Migrating from 0.9
+
+Genkit 1.0 introduces some of breaking changes alongside feature enhancements that improve overall functionality. If you have been developing applications with Genkit 0.9, you will need to update your application code when you upgrade to the latest version. This guide outlines the most significant changes and offers steps to migrate your existing applications smoothly.
+If you're using 0.5, use <> to upgrade to 0.9 first before upgrading to 1.0
+
+
+## Streaming functions do not require an await
```

Comment:
I suggest ordering these in most largest conceptual order first, so I'd start with the Beta API before the more point-wise changes.